src/index.js

```diff
@@ -53,6 +54,10 @@ const enableRequestLogging = ENABLE_REQUEST_LOGGING === "true"
 const logQueryDetailsThreshold =
   LOG_QUERY_DETAILS_THRESHOLD && parseInt(LOG_QUERY_DETAILS_THRESHOLD, 10) // null by default
 
+if (NODE_ENV !== "test") {
```

Comment:
Perhaps add a boolean config value such as `config.HEAP_DUMPS_ENABLED` and only import if it is enabled?  Would be good to isolate this if it has the potential to impact runtime performance if imported but unused.